packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -338,7 +333,8 @@ export async function sideloadAddIn(manifestPath: string, app?: OfficeApp, canPr
       await registerAddIn(manifestPath);
       // for Outlook, open Outlook.exe; for other Office apps, open the document
       if (app == OfficeApp.Outlook) {
-        if (!(await hasOutlookVersion("16.0.13709"))) {
+        const version = await getOutlookVersion();
+        if (!hasOfficeVersion("16.0.13709", version)) {
```

Comment:
Do the check about whether version is undefined here.

It would be a good practice to put type info on the version variable, e.g. `const version: string | undefined = `
It will still be implicit from the return of getOutlookVersion().
You don't want to be able to pass undefined into hasOfficeVersion(). Rather just check here.

`if (version && !hasOfficeVersion())`

In this specific case, we want the fact that if the version cannot be retrieved we still allow it to sideload.

Reply:
Sure, thanks for the feedback!
Just made those changes :D

So, in general, we like to treat the undefined outside a function?